src/routes/Profile/Competences/constants.ts

```diff
@@ -12,16 +11,17 @@ export interface FilterData {
   checked: boolean;
 }
 
-export type FiltersType = Record<OsaaminenLahdeTyyppi, FilterData[]>;
-export const FILTERS_ORDER = ['TOIMENKUVA', 'KOULUTUS', 'PATEVYYS', 'KIINNOSTUS', 'MUU_OSAAMINEN'] as const;
-export const COMPETENCE_TYPES = ['TOIMENKUVA', 'KOULUTUS', 'PATEVYYS', 'MUU_OSAAMINEN'] as const;
+export const FILTERS_ORDER = ['TOIMENKUVA', 'KOULUTUS', 'PATEVYYS', 'MUU_OSAAMINEN'] as const;
```

Comment:
Name could be better, but this reminds that the order matters in this array how they are shown in the Profile/Competences filter box.